benchmark_v2/run_benchmarks.py

```diff
@@ -237,8 +239,8 @@ def upload_results_to_hf_dataset(
     logger.info(f"Uploading benchmark results to dataset '{dataset_name}' at path '{repo_path}'")
 
     try:
-        # Get the authentication token (prioritize specific token, fallback to HF_TOKEN)
-        token = os.getenv("TRANSFORMERS_CI_RESULTS_UPLOAD_TOKEN") or os.getenv("HF_TOKEN")
+        # Get the authentication token (prioritize passed token, then env vars)
+        auth_token = token or os.getenv("HF_TOKEN")
```

Comment:
I might be wrong, but I think we don't need to specify `os.getenv("HF_TOKEN")` here.

If `token` is `None`, the `huggingface_hub` will take care of using `HF_TOKEN`.

Fine fore me however if you want to keep as it is.